src/datachain/client/fsspec.py

```diff
@@ -211,7 +211,7 @@ async def get_current_etag(self, file: "File") -> str:
         info = await self.fs._info(
             self.get_full_path(file.path, file.version), **kwargs
         )
-        return self.info_to_file(info, "").etag
+        return self.info_to_file(info, file.path).etag
```

Comment:
This param is not used here, since we are only taking `etag`, but for consistency why not to pass file path here?